site/hostapps.html

```diff
@@ -226,12 +223,16 @@
   </div>
 
   <div class="legend">
-    <br>* Buffer Override, EQ Sync, Scrubby, and Skidder have tempo sync features
+    <br>&loz; Buffer Override, EQ Sync, Scrubby, and Skidder have tempo sync features
     <br>&dagger; Buffer Override, Geometer, MIDI Gater, Rez Synth, Scrubby, Skidder, and Transverb are effects that process audio and MIDI
     <br>&sect; Turntablist is an "instrument" plugin
-    <br>&loz; = free
-    <br>&#9734; forthcoming version (not available yet)
+    <br>* = free
+    <p>
+      We do not maintain this list with much diligence. 
+      <br>You may find the software search tool at <a href="https://www.kvraudio.com">KVR Audio</a> to be more useful.
```

Comment:
All for being helpful but it wasn't clear which part you're recommending... is it https://www.kvraudio.com/q.php ?

Reply:
yes!   Okay I honed the link to yours.